src/containers/tud/TimeUseDiaryContainer.test.js

```diff
@@ -0,0 +1,154 @@
+// @flow
+
+import React, { Suspense } from 'react';
+
+import i18n from 'i18next';
+import { mount } from 'enzyme';
+import { Select } from 'lattice-ui-kit';
+import { act } from 'react-dom/test-utils';
+// $FlowFixMe
+import { I18nextProvider, initReactI18next } from 'react-i18next';
+import { Provider } from 'react-redux';
+import {
+  BrowserRouter,
+  Redirect,
+  Route,
+  Switch
+} from 'react-router-dom';
+
+import ConfirmChangeLanguage from './components/ConfirmChangeLanguage';
+import QuestionnaireForm from './components/QuestionnaireForm';
+import SupportedLanguages from './constants/SupportedLanguages';
+import SurveyIntro from './components/SurveyIntro';
+import TimeUseDiaryContainer from './TimeUseDiaryContainer';
+import TranslationKeys from './constants/TranslationKeys';
+
+import Translations from '../../core/i18n/translations';
+import initializeReduxStore from '../../core/redux/ReduxStore';
+import initializeRouterHistory from '../../core/router/RouterHistory';
+import * as LanguageCodes from '../../utils/constants/LanguageCodes';
+import * as Routes from '../../core/router/Routes';
+
+const resources = {
+  en: {
+    translation: Translations.en
+  }
+};
+
+i18n
+  .use(initReactI18next)
+  .init({
+    lng: LanguageCodes.ENGLISH,
+    fallbackLng: LanguageCodes.ENGLISH,
+    resources
+  });
+
+const routerHistory = initializeRouterHistory();
+const reduxStore = initializeReduxStore(routerHistory);
+
+const App = (
+  <Suspense fallback="...">
+    <Provider store={reduxStore}>
+      <I18nextProvider i18n={i18n}>
+        <BrowserRouter>
+          <Switch>
+            <Route path={Routes.TUD} component={TimeUseDiaryContainer} />
+            <Redirect to={Routes.TUD} />
+          </Switch>
+        </BrowserRouter>
+      </I18nextProvider>
+    </Provider>
+  </Suspense>
+);
+
+const testIfMultipleLanguages = (languages) => (languages.length > 1 ? test : test.skip);
```

Comment:
This doesn't seem necessary because we'll always have more than 1 supported language. You even translated everything for testing purposes.